google-youtube.html

```diff
@@ -1,637 +1,637 @@
-              // The YouTube Player API only exposes playback data about a video once
-              // playback has begun.
-              if (this.state == 1) {
-                this.playbackstarted = true;
-
-                // After playback has begun, play() can always be used to resume playback if the video is paused.
-                this.playsupported = true;
-
-                this.duration = this.player.getDuration();
-                this.durationformatted = this.toHHMMSS(this.duration);
-
-                if (!this.updatePlaybackStatsInterval) {
-                  this.updatePlaybackStatsInterval = setInterval(this.updatePlaybackStats.bind(this), 1000);
-                }
-              } else {
-                // We only need to update the stats if the video is playing.
-                if (this.updatePlaybackStatsInterval) {
-                  clearInterval(this.updatePlaybackStatsInterval);
-                  this.updatePlaybackStatsInterval = null;
-                }
-              }
-
-              this.fire('google-youtube-state-change', e);
-            }.bind(this),
-            onError: function(e) {
-              // Set the player state to 0 ('ended'), since playback will have stopped.
-              this.state = 0;
-
-              this.fire('google-youtube-error', e);
-            }.bind(this)
-          }
-        });
-      },
-
-      updatePlaybackStats: function() {
-        this.currenttime = Math.round(this.player.getCurrentTime());
-        this.currenttimeformatted = this.toHHMMSS(this.currenttime);
-        this.fractionloaded = this.player.getVideoLoadedFraction();
-      },
-
-      toHHMMSS: function(totalSeconds) {
-        var hours = Math.floor(totalSeconds / 3600);
-        totalSeconds -= hours * 3600;
-        var minutes = Math.floor(totalSeconds / 60);
-        var seconds = Math.round(totalSeconds - (minutes * 60));
-
-        var hourPortion = '';
-        if (hours > 0) {
-          hourPortion += hours + ':';
-
-          if (minutes < 10) {
-            minutes = '0' + minutes;
-          }
-        }
-
-        if (seconds < 10) {
-          seconds = '0' + seconds;
-        }
-
-        return hourPortion + minutes + ':' + seconds;
-      },
-
-      handleThumbnailTap: function() {
-        this.autoplay = 1;
-        this.thumbnail = '';
-      }
-    });
-  </script>
-</polymer-element>
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../core-localstorage/core-localstorage.html">
+<link rel="import" href="../google-apis/google-youtube-api.html">
+
+<!--
+`google-youtube` encapsulates the YouTube player into a web component.
+
+    <google-youtube
+      videoid="..."
+      height="270px"
+      width="480px"
+      rel="0"
+      start="5"
+      autoplay="1">
+    </google-youtube>
+
+`google-youtube` supports all of the [embedded player parameters](https://developers.google.com/youtube/player_parameters). Each can be set as an attribute on `google-youtube`.
```

Comment:
I don't know why this happened to be honest.

I just cloned the repository by using this following command: `git clone`.

Maybe I'm doing something wrong? Any advice?

Thanks.